uv.gyp

```diff
@@ -92,7 +94,7 @@
           '-Wno-unused-parameter',
           '-Wstrict-prototypes',
         ],
-        'OTHER_CFLAGS': [ '-g', '--std=gnu89', '-pedantic' ],
+        'OTHER_CFLAGS': [ '-g', '--std=gnu89' ],
```

Comment:
should this be gnu99, since we switched to c99 in master?

Reply:
Yeah, unless we want to change master back to c89. I don't really want to combine this change into the merge commit though, so I'll add it to my bulleted list of independent fixups above.

Reply:
Ack, thanks!